tests/unit/LedgerConsentContinuationOut.spec.ts

```diff
@@ -4,6 +4,10 @@ import { mount } from '@vue/test-utils'
 import ConsentContinuationOut from '@/components/Dashboard/FilingHistoryList/filings/ConsentContinuationOut.vue'
 import { createPinia, setActivePinia } from 'pinia'
 
+// mock the console.warn function to hide "[Vuetify] The v-expansion-panel component must be used inside a
+// v-expansion-panels"
+console.warn = jest.fn()
+
```

Comment:
As mentioned earlier, I spent some time trying to fix this using `parentComponent` but bumped into other issues. Not worth a lot more time to fix this (IMHO).